src/include/metrics/query_trace_metric.h

```diff
@@ -92,10 +98,18 @@ class QueryTraceMetricRawData : public AbstractRawData {
   };
 
   struct QueryTrace {
-    QueryTrace(const execution::query_id_t query_id, const uint64_t timestamp)
-        : query_id_(query_id), timestamp_(timestamp) {}
+    QueryTrace(const execution::query_id_t query_id, const uint64_t timestamp, std::string param_string,
+               std::string type_string, const catalog::db_oid_t oid)
+        : query_id_(query_id),
+          timestamp_(timestamp),
+          param_string_(std::move(param_string)),
+          type_string_(std::move(type_string)),
+          oid_(oid) {}
     const execution::query_id_t query_id_;
     const uint64_t timestamp_;
+    std::string param_string_;
```

Comment:
These should be `const` like the other fields.

Reply:
Resolved in the latest push.